homeassistant/components/home_connect/light.py

```diff
@@ -194,7 +194,7 @@ async def async_update(self) -> None:
 
         else:
             brightness = self.device.appliance.status.get(self._brightness_key, {})
-            if brightness is None:
+            if not brightness:
```

Comment:
Is 0 a valid value for brightness? if it is, it would be better to change:
`brightness = self.device.appliance.status.get(self._brightness_key)` 
and keep the original condition

Reply:
I guess 0 is not a valid value since if the value from `self.device.appliance.status.get(self._brightness_key)` is returned correctly it's used later as `brightness.get(ATTR_VALUE)` which would fail if brightness was an integer. Isn't it?

Reply:
I think that the brightness variable is not what you think. There are two cases, if `self.device.appliance.status.get(self._brightness_key, {})` returns a valid value or if the `self.device.appliance.status` dict doesn't have a `self._brightness_key` key and it would return the default value (`{}`).

- If the returned value is valid, then that's handled in line 200 (just a few lines below) and  `brightness.get(ATTR_VALUE)` would be executed. This means that in this case, brightness is a dict which would have an ATTR_VALUE key.

- If the returned value is not valid, then an empty dict would be assigned to brightness, which is the reason the check for `if brightness is None` was wrong and I changed it to `if not brightness`.